lib/pbench/cli/server/pbench_create_user.py

```diff
@@ -0,0 +1,94 @@
+import click
+from pbench import BadConfig
+from pbench.server import PbenchServerConfig
+from pbench.cli.server import pass_cli_context
+from pbench.cli.server.options import common_options
+from pbench.common.logger import get_pbench_logger
+from pbench.server.database.models.users import User
+from pbench.server.database.database import Database
+
+_NAME_ = "pbench-create-user"
+
+
+class PbenchCreateUser:
+    def __init__(self, context):
+        self.context = context
+
+    def execute(self):
+        try:
+            config = PbenchServerConfig(self.context.config)
+        except BadConfig:
+            raise
```

Comment:
I guess I won't push to reproduce the entire agent command support hierarchy here, but I think the agent-side `BaseCommand` (in `lib/pbench/agent/base.by`, which centralizes this sort of setup in one place) is a good idea, and we should be duplicating that on the server side and inheriting from that for our utility commands instead of duplicating common code like this.

And, yeah, once we get this working I should retrofit my utility commands to fit. But I'll leave it to you whether to take that step now or wait for someone to refactor it later... ;-)